### PR TITLE
Upgrade Helm to v3.4.0

### LINKED ISSUE #11: Helm test doesn't use Zap logger
Helm test logs write to stdout directly,  we should use Zap as the test runner logger.

```
Pod frontend-podinfo-grpc-test-d0acu pending
Pod frontend-podinfo-grpc-test-d0acu running
{"level":"info","ts":1594408916.3492382,"logger":"controllers.HelmRelease","msg":"Add/Modify event for frontend-podinfo-grpc-test-d0acu: MODIFIED","helmrelease":"helm-system/frontend"}
{"level":"info","ts":1594408917.3560367,"logger":"controllers.HelmRelease","msg":"Add/Modify event for frontend-podinfo-grpc-test-d0acu: MODIFIED","helmrelease":"helm-system/frontend"}
```

### PR DESCRIPTION
**Pull request by hiddeco**
This Helm version includes a fix to properly log the output of Pod
status phases for (test) jobs.

Fixes #11

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
Yey finally structured logs everywhere 🎉

**Comment by hiddeco** [comment]
Due to Helm depending on API packages made available in `v1.19.x`; `v1.19.x` depending on `github.com/go-logr/logr` `v0.2.0`, and `sigs.k8s.io/controller-runtime` depending on `logr.InfoLogger` that is no longer available since `>=v0.2.0`.

We are either forced to upgrade to the latest `v0.7.0` RC for `sigs.k8s.io/controller-runtime`, or need to wait for the official release before this can be merged.

**Pull request merged**
